devdoc/constbuffer_requirements.md

```diff
@@ -172,6 +172,8 @@ Given an existing `handle` `CONSTBUFFER_CreateFromOffsetAndSize` creates another
 
 **SRS_CONSTBUFFER_02_027: [** If `offset` + `size` exceed `handles`'s size then `CONSTBUFFER_CreateFromOffsetAndSize` shall fail and return `NULL`. **]**
 
+If `offset` is 0 and `size` is equal to `handle`'s size then `CONSTBUFFER_CreateFromOffsetAndSize` shall return `handle`.
```

Comment:
>`CONSTBUFFER_CreateFromOffsetAndSize` shall return `handle`. [](http://example.com/codeflow?start=61&length=60)

I think we also have to specify that we increment the reference count.

Reply:
Done